Doc/tutorial/inputoutput.rst

```diff
@@ -329,11 +329,16 @@ equivalent :keyword:`try`\ -\ :keyword:`finally` blocks::
 
 If you're not using the :keyword:`with` keyword, then you should call
 ``f.close()`` to close the file and immediately free up any system
-resources used by it. If you don't explicitly close a file, Python's
-garbage collector will eventually destroy the object and close the
-open file for you, but the file may stay open for a while.  Another
-risk is that different Python implementations will do this clean-up at
-different times.
+resources used by it.
+
+.. warning::
+   Calling ``f.write()`` without using the :keyword:`!with` keyword or calling
+   ``f.close()``, ``f.flush()`` or ``del f`` **might** result in the arguments
```

Comment:
```suggestion
   ``f.close()`` **might** result in the arguments
```

``del f`` doesn't guarantee that ``__del__`` is called. So it should be removed.
And I suggest to remove `f.flush()` too because:

* Here, the tutorial teaching `with` and `f.close()`. `f.flush()` is not described yet.
* Even if `f.flush()` is called, not closing file is bad idea.
* No need to cover how to flush buffered data here. Otherwise, we need to list up all methods causing flush.

Reply:
If you think so...